cypress/integration/console/shared/payload-formatters/edit.spec.js

```diff
@@ -0,0 +1,525 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Payload formatters', () => {
+  const applicationId = 'test-application-payload-formatters'
+  const application = { ids: { application_id: applicationId } }
+  const userId = 'edit-app-payload-formatter-test-user'
+  const user = {
+    ids: { user_id: userId },
+    primary_email_address: 'edit-application-formatters-test-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+
+  before(() => {
+    cy.dropAndSeedDatabase()
+    cy.createUser(user)
+  })
+
+  describe('Application', () => {
+    before(() => {
+      cy.loginConsole({ user_id: userId, password: user.password })
+      cy.createApplication(application, userId)
+      cy.clearLocalStorage()
+      cy.clearCookies()
```

Comment:
Is this really necessary? Cypress should clear cookies and localStorage between tests anyway.

[https://docs.cypress.io/api/commands/clearlocalstorage.html#Syntax](https://docs.cypress.io/api/commands/clearlocalstorage.html#Syntax)

https://docs.cypress.io/api/commands/clearcookies.html#Syntax

See also my comment below. I think we can remove this before hook altogether.